sycl/source/detail/device_info.hpp

```diff
@@ -228,28 +228,16 @@ struct get_device_info_impl<std::vector<info::fp_config>,
   }
 };
 
-inline bool checkNativeQueueProfiling(const DeviceImplPtr &Dev) {
-  pi_queue_properties Properties;
-  Dev->getPlugin()->call<PiApiKind::piDeviceGetInfo>(
-      Dev->getHandleRef(), PiInfoCode<info::device::queue_profiling>::value,
-      sizeof(Properties), &Properties, nullptr);
-  return Properties & PI_QUEUE_FLAG_PROFILING_ENABLE;
-}
-
 // Specialization for queue_profiling. In addition to pi_queue level profiling,
-// piGetDeviceAndHostTimer support is needed for command_submit query support.
+// piGetDeviceAndHostTimer is not supported, command_submit, command_start,
+// command_end will be calculated. see MFallbackProfiling
```

Comment:
This wasn't fixed at all, but the conversation was resolved, @eunakim0103 
https://github.com/intel/llvm/pull/10662#discussion_r1291180455